DOCUMENTATION.md

```diff
@@ -1,26 +1,38 @@
-## Table of Contents
+# Table of Contents
 * [Description](#description)
 * [MQTTClient](#mqttclient)
```

Comment:
Why have two separate clients?

Reply:
For now you have two clients, one for mqtt the other to deal with applications. The next step is what I have been working on to follow the path of what's done in the Node SDK, a global handler which will then allow the creation of an application client or an mqtt client. This in currently on branch feature/handler-client which I will rebase once I've done the needed correction for the application client feature

Reply:
I'd be in favor of waiting until there's a global handler ready, to avoid confusion for SDK users when upgrading from a version to another.

Reply:
So once all changes are done in this branch, should I rebase and then create a PR to merge the handler-client branch into this one?

Reply:
I think you should merge the `handler-client` branch into this one!